testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/preload/PreloadMiniAppWindow.kt

```diff
@@ -40,11 +41,12 @@ class PreloadMiniAppWindow(
     @Suppress("LongParameterList")
     fun initiate(
         appInfo: MiniAppInfo?,
-        miniAppId: String,
-        versionId: String,
+        miniAppIdAndVersionIdPair: Pair<String, String>,
```

Comment:
was this change required? If yes, should we create a more explicit dedicated data class?

Reply:
In the sense of mistook the `first` and the `second` written by devs, yeah +1